ci-operator/config/openshift/verification-tests/openshift-verification-tests-master__installer-rehearse-4.12.yaml

```diff
@@ -104,6 +104,11 @@ tests:
   steps:
     cluster_profile: ibmcloud-qe
     workflow: cucushift-installer-rehearse-ibmcloud-ipi
+- as: installer-rehearse-ibmcloud-private
```

Comment:
You can directly replace `workflow: cucushift-installer-rehearse-ibmcloud-ipi` line with your new workflow to run installer rehearse job, not have to introduce a new one.